File: Java/app/src/main/java/dev/ogabek/java/model/Filter.java
package dev.ogabek.java.model;

public class Filter {

    private String title = null;
    private boolean isExploreHave = false;
    private final boolean isSelected;

    public Filter(boolean isExploreHave, boolean isSelected) {
        this.isExploreHave = isExploreHave;
        this.isSelected = isSelected;
    }
    public Filter(String title, boolean isSelected) {
        this.title = title;
        this.isSelected = isSelected;
    }

    public String getTitle() {
        return title;
    }

    public boolean isSelected() {
        return isSelected;
    }

    public boolean isExploreHave() {
        return isExploreHave;
    }
}

File: Kotlin/app/src/main/java/dev/ogabek/kotlin/model/Filter.kt
package dev.ogabek.kotlin.model

class Filter {
    var title: String? = null
        private set
    var isExploreHave = false
        private set
    val isSelected: Boolean

    constructor(isExploreHave: Boolean, isSelected: Boolean) {
        this.isExploreHave = isExploreHave
        this.isSelected = isSelected
    }

    constructor(title: String?, isSelected: Boolean) {
        this.title = title
        this.isSelected = isSelected
    }
}